### DISCUSSION TITLE
Is React Spectrum compatible with Preact?

### DISCUSSION THREAD

**Discussion by vxncetxn** [adobe/react-spectrum#2498] [Q&A]
Hello, just wondering whether React Spectrum libraries are fully/partially/not compatible with Preact. I was actually using another accessible-by-default component library (Radix UI) previously but it is not compatible with Preact at all and their maintainers told me that Preact compatibility is a non-goal for them so I am just wondering what to expect here.

**Comment by reidbarber** [comment]
There is an issue for adding Preact support:
- https://github.com/adobe/react-spectrum/issues/781

Check the comments for a list of issues that came up when testing.
